bin/utils/idam-create-service.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -eu
+
+dir=$(dirname ${0})
+
+LABEL=${1}
+CLIENT_ID=${2}
+CLIENT_SECRET=${3}
+REDIRECT_URL=${4}
+SELF_REGISTRATION=${5:-"false"}
+SCOPE=${6:-"openid profile authorities acr roles search-user"}
```

Comment:
please change default to "openid profile roles" to inline with IDAM.

Reply:
https://github.com/hmcts/ccd-docker/pull/103/commits/2a11ab59294f83f9ce45b5e51cda3ea2bc309264